build/scripts/VerifyExecutePermissions.ps1

```diff
@@ -0,0 +1,91 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $ModulesDirectory
+)
+
+Set-StrictMode -Version Latest
+Write-Host "Verifying execute permissions on all objects in $ModulesDirectory"
+
+$includefilter = @('*.Table.al', '*.Report.al', '*.Codeunit.al', '*.Page.al', '*.XMlPort.al', '*.Query.al')
+$AlObjects = @{}
+
+foreach ($folder in (Get-ChildItem -path $ModulesDirectory -Directory)) {

```

Comment:
I don't get why it's going to every folder in a module. Shouldn't it just scan recursively for .al files in the module's folder?

Reply:
It probably should 😆 Let me adjust that